### DISCUSSION TITLE
MQTT statestream with discovery config

### DISCUSSION THREAD

**Discussion by koying** [home-assistant/architecture#670] [General] [answered]
Hi,

I've created a custom component that "augment" `mqtt_statestream` by also creating discovery entries for the entities that are published to MQTT: https://github.com/koying/mqtt_discoverystream_ha

The use-case is master-slave HA configurations via MQTT

This to explore if there is any interest to implement that functionality in `mqtt_statestream` core itself.

**Comment by frenck** [comment, marked as answer]
This idea didn't gain enough traction in recent years and is running stale.

I'm going to close it for now; we can reconsider it in the future if this idea becomes relevant again.

../Frenck
